### ISSUE TITLE
something went wrong with my own dataset in regression

### ISSUE THREAD

**Issue by P22222** [online-ml/river#854]
Hi all, I'm a online-ml starter, and happy to see such a good tool to launch a online learning project. But I exactly encountered some problems and dont know how to fix it, I turned to google with no result. 
In fact, I'm working on a online ml project with regression problem. I use lightgbm and randomforestRegressor in sklearn, which have good performance on my dataset, but obviously, they have no online version. So I tried some of regressor class in river, like HoeffdingTreeRegressor,HoeffdingAdaptiveTreeRegressor,LinearRegression,SGTRegressor and so on.But they get ridiculous results, y_true range from 9-12 mostly while y_predict can reach 1e5 even higher and sometimes negative,or go wrong (like SGTRegressor I captured a _'float division by zero'_ Error) and I dont know why, can you tell me what went wrong and how to fix them.

Fig1. Loss 
![image](https://user-images.githubusercontent.com/24954798/155055894-309c9454-f08a-46a2-893c-22f67a65bccb.png)

Fig2. y_pred(3 models) and y_true
![Figure_1](https://user-images.githubusercontent.com/24954798/155056378-8b460e05-3ad3-4969-a974-23df73d2b917.png)

Fig3. LightGBM performance
![LightGBM](https://user-images.githubusercontent.com/24954798/155056175-cd1fc154-a67b-4658-b569-b185f6713ae2.png)

Fig4. RFRegressor Performance
![RandomForest](https://user-images.githubusercontent.com/24954798/155056193-0b1ddf32-d0d5-4683-a270-473e491bab22.png)

**Comment by MaxHalford** [comment]
Hello. We can't help you without a dataset and a reproducible piece of code.

**Comment by P22222** [comment]
Thank you for your replying. I've made a demo-illustration. How can I send you personally?

**Comment by MaxHalford** [comment]
You can send it by email to maxhalford25@gmail.com.

**Comment by P22222** [comment]
Thank u sir. I've sent you a E-mail named _mini-dataset & code for Issue 854_ , forward to your reply. Give you my thankfulness again.

**Comment by MaxHalford** [comment]
I received the email, I'll check it out soonish :)

**Comment by MaxHalford** [comment]
Mmm actually I never seemed to have received this email, which is why I never got back to you. I'll close this, but please let me know if you still want me to take a look. Cheers!
